mmaction/models/recognizers/base.py

```diff
@@ -154,9 +160,16 @@ def _parse_losses(losses):
 
         return loss, log_vars
 
-    def forward(self, imgs, label=None, return_loss=True, **kwargs):
+    def forward(self,
+                imgs,
+                label=None,
+                gradcam=False,
```

Comment:
I prefer not to add too many custom features in BaseClass, maybe catch `gradcam` by:
```python
        # if gradcam:
        if kwargs.get(gradcam, False)
            return self.forward_gradcam(imgs, **kwargs)
```
Since we already allow kwargs.